.github/workflows/publish.yml

```diff
@@ -91,3 +94,22 @@ jobs:
           asset_path: ./${{ matrix.asset_name }}.gz
           asset_name: ${{ matrix.asset_name }}.gz
           asset_content_type: application/gzip
+
+      - name: Generate asset hash
+        run: ${{ matrix.shasum_cmd }} ${{ matrix.asset_name }}.gz | awk '{ print $1 }' > ${{ matrix.asset_name }}.gz.sha256
```

Comment:
Both shasum and sha256sum output in this format: `<computed hash> <filename>`. So `shasum <filename> | awk '{ print $1 }'` will just output the hash